clipdel

```diff
@@ -40,10 +41,13 @@ if ! [[ -f $cache_file ]]; then
     exit 0  # Well, this is a kind of success...
 fi
 
-# https://github.com/koalaman/shellcheck/issues/1141
-# shellcheck disable=SC2124
-raw_pattern=$1
-esc_pattern=${raw_pattern//\#/'\#'}
+if [[ -n $1 ]]; then
+    raw_pattern=$1
+elif ! [[ -t 0 ]]; then
+    IFS= read -r raw_pattern
+fi
+
+esc_pattern=${raw_pattern/\#/'\#'}
```

Comment:
Hmm, why only one? We want to update all of them in the string, no?

Reply:
Well spotted, thanks! It's an unintended change. At the time I forked, clipdel was still only escaping the first one, and I missed the single-character change in this line during rebase.

Fixed.